products/cpdb/bash/01_preprocessing.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+source bash/config.sh
+
+run_sql_file "sql/_create.sql"
+
+echo "fixing dot_bridges"
+run_sql_command "ALTER TABLE dot_projects_bridges RENAME COLUMN fmsid TO fms_id;"
+python3 python/dot_bridges.py
+
+echo "fixing doitt_buildingfootprints"
+run_sql_command "ALTER TABLE doitt_buildingfootprints RENAME TO doitt_buildingfootprints_source;"
```

Comment:
Fyi, you can just add an "import_as: doitt_buildingfootprints_source" to the recipes